### PR TITLE
WinUI-2482_UG_Document_for_Leading_and_Trailing_View

### PR DESCRIPTION
**Pull request by YuvarajMasilamani**
1.UG document for leading and trailing view in AutoComplete .
2.UG document for leading and trailing view in ComboBox .

### CODE REVIEW

**Comment by jeganraj-m** [comment]
rebuild

**Comment by jeganraj-m** [comment]
rebuild

File: winui/AutoComplete/Leading-and-Trailing-view.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+title: Leading and Trailing View with WinUI AutoComplete | Syncfusion
+description: Learn here all about Leading and Trailing View support in Syncfusion WinUI AutoComplete control and more.
+platform: WinUI
+control: SfAutoComplete
+documentation: ug
+---
+
+# Leading  and Trailing View in WinUI AutoComplete (SfAutoComplete)
+
+This section explains about the Leading and Trailing view support available in [AutoComplete](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfAutoComplete.html).The `Leadingview` appears before the `AutoComplete` selection area and `Trailingview` appears after the `AutoComplete` selection area. Any content, such as an icon, image, button, or other control, can be displayed in the `Leadingview` and `Trailingview`.
```
Comment (jeganraj-m) [inline winui/AutoComplete/Leading-and-Trailing-view.md:12]: Any content, such as an icon, image, button, or control,

File: winui/AutoComplete/Leading-and-Trailing-view.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                        <FontIcon Glyph="&#xEBE7;" />
+                    </Viewbox>
+              </Button>
+        </editors:SfAutoComplete.TrailingView>
+</editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Trailing View in WinUI AutoComplete](Leading_and_Trailing_view_images/Trailing-View-in-AutoComplete.png)
```
Comment (jeganraj-m) [inline winui/AutoComplete/Leading-and-Trailing-view.md:61]: WinUI AutoComplete control with button in trailing view
WinUI AutoComplete control with icon in leading view
WinUI AutoComplete control with combo box in leading view and button in trailing view

File: winui/AutoComplete/Leading-and-Trailing-view.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Both Leading and Trailing View
```
Comment (jeganraj-m) [inline winui/AutoComplete/Leading-and-Trailing-view.md:63]: Leading and Trailing View

File: winui/AutoComplete/Leading-and-Trailing-view.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Leading and Trailing View with WinUI AutoComplete | Syncfusion
+description: Learn here all about Leading and Trailing View support in Syncfusion WinUI AutoComplete control and more.
```
Comment (jeganraj-m) [inline winui/AutoComplete/Leading-and-Trailing-view.md:4]: Learn how to add leading and trailing view in Syncfusion WinUI AutoComplete control and more.

File: winui/AutoComplete/Leading-and-Trailing-view.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+title: Leading and trailing view in WinUI AutoComplete | Syncfusion
+description: Learn how to add leading and trailing view (front and back view) in Syncfusion WinUI AutoComplete control and more.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# Leading and Trailing View in WinUI AutoComplete (SfAutoComplete)
+
+This section explains about the leading and trailing view support available in [AutoComplete](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfAutoComplete.html).The `LeadingView` appears before the `AutoComplete` selection area and `TrailingView` appears after the `AutoComplete` selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView`.
```
Comment (florenceliliana) [inline winui/AutoComplete/Leading-and-Trailing-view.md:12]: This section explains about the leading and trailing view support available in [AutoComplete](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfAutoComplete.html). The `LeadingView` appears before the `AutoComplete` selection area and `TrailingView` appears after the `AutoComplete` selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView.`

File: winui/ComboBox/Leading-and-Trailing-View.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+title: Leading and trailing view in WinUI ComboBox control | Syncfusion
+description: Learn how to add leading and trailing view (front and back view) in Syncfusion WinUI ComboBox control and more.
+platform: winui
+control: SfComboBox
+documentation: ug
+---
+
+# Leading and Trailing View in WinUI ComboBox (SfComboBox)
+
+This section explains about the leading and trailing view support available in [ComboBox](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfComboBox.html).The `LeadingView` appears before the `ComboBox` selection area and `TrailingView` appears after the `ComboBox`  selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView`.
```
Comment (florenceliliana) [inline winui/ComboBox/Leading-and-Trailing-View.md:12]: This section explains about the leading and trailing view support available in [ComboBox](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfComboBox.html). The `LeadingView` appears before the `ComboBox` selection area, and `TrailingView` appears after the `ComboBox` selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView.`

**Review by florenceliliana — CHANGES_REQUESTED** [review]

**Review by jeganraj-m — APPROVED** [review]

**Pull request merged**